api/src/main/java/org/pmiops/workbench/db/model/DbUserTermsOfService.java

```diff
@@ -0,0 +1,75 @@
+package org.pmiops.workbench.db.model;
+
+import java.sql.Timestamp;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+
+@Entity
+@Table(name = "user_terms_of_service")
+public class DbUserTermsOfService {
+  private long userTermsOfServiceId;
+  private long userId;
+  private int tosVersion;
+  private Timestamp agreementTime;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "user_terms_of_service_id")
+  public long getUserTermsOfServiceId() {
+    return userTermsOfServiceId;
+  }
+
+  public void setUserTermsOfServiceId(long userTermsOfServiceId) {
+    this.userTermsOfServiceId = userTermsOfServiceId;
+  }
+
+  @Column(name = "user_id", nullable = false)
+  public long getUserId() {
```

Comment:
Was it necessary to change the User model to a long? At some point, we should decide which access pattern we want to standardize on. I think there's good arguments for both approaches.